docs/stack/_index.md

```diff
@@ -30,6 +30,10 @@ Redis Stack was created to allow developers to build real-time applications with
 
 Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles five Redis modules: [RedisJSON](/docs/stack/json), [RediSearch](/docs/stack/search), [RedisGraph](/docs/stack/graph), [RedisTimeSeries](/docs/stack/timeseries), and [RedisBloom](/docs/stack/bloom).
 
+### Use cases
+
+Explore [use cases](/docs/stack/use-cases/) of the applied modern data models and processing engines provided by Redis Stack across industries and applications.
```

Comment:
Do we detail use cases for the processing engines, or just for the modules?

Reply:
Modern data models? Is that a marketing term?

Reply:
Most likely is. What's your take on how we should word it?

Reply:
Live it as is. I see that we also use this term in https://redis.com/modules/get-started/